validator.py

```diff
@@ -0,0 +1,117 @@
+import configparser
```

Comment:
This is the library file from the test-harness and is already in this repo: `__init__.py`

Please remove this file.

Reply:
That's because the PR is against master. If we change it to be based on ode-1227, it will be only one file changed.